airbyte-workers/src/main/java/io/airbyte/workers/general/DefaultCheckConnectionWorker.java

```diff
@@ -88,10 +90,8 @@ public StandardCheckConnectionOutput run(final StandardCheckConnectionInput inpu
       }
 
     } catch (final Exception e) {
-      LOGGER.error("Error while checking connection: ", e);
-      return new StandardCheckConnectionOutput()
-          .withStatus(Status.FAILED)
-          .withMessage("Error while getting checking connection, because of: " + e.getMessage());
+      LOGGER.error("Unexpected error while checking connection: ", e);
+      throw new WorkerException("Unexpected error while getting checking connection.", e);
```

Comment:
Do we need this log and exception wrap? Feels like we end up duplicating error logs.

Reply:
I think it will be preferable for the user support team to see the ERROR tag. I might duplicate the log of the error but I would prefer that since it ensure that the error log and the root exception will be call in the same place.

Reply:
I wonder if we should come up with a better labeling in this use case. That would make the search easier than scanning for `error` if we know it is an error relevant for them.